app/views/resorts/_forecast_overview.html.erb

```diff
@@ -64,7 +64,7 @@
   <% resort.forecasts.each.with_index do |forecast, index| %>
     <div class="summary-card" id="<%= forecast.condition %>">
       <h1> <%= days[index].upcase %> </h1>
-      <h2> <%= forecast.condition.upcase %> </h2>
```

Comment:
this bug causing the page to break..since small change, included in the same working branch @swmoore84